modules/v2/variables.tf

```diff
@@ -59,10 +59,7 @@ variable "containers" {
       container_port = optional(number, 8080)
     }), {})
     resources = optional(object({
-      limits = optional(object({
-        cpu    = optional(string)
-        memory = optional(string)
-      }))
+      limits = optional(map(string))
```

Comment:
Can we add gpu here instead of making this map(string). This would be much more clear to users while configuring.